### PR TITLE
Update scan radius

### PR DESCRIPTION
**Pull request by asottile**
This also adds a nifty overlay to visualize the scan radius.  Simply add `/?areas` (or `&areas`) to the url:

### CODE REVIEW

**Comment by asottile** [comment]
@bukzor: I think you'll find the visual useful ^

**Comment by bukzor** [comment]
Where did the numbers come from? The minus one is particularly odd.

File: scrape.py [hunk 1]
```diff
@@ -363,7 +365,7 @@ def generate_location_steps3(num_steps):
 
 
 def generate_location_steps4(loc, num_steps):
-    meters_delta = 150
+    meters_delta = RADIUS * math.sqrt(3) - 1
```
Comment (bukzor) [inline scrape.py:368]: RADIUS \* UNIT_TRIANGLE_HEIGHT \* 2  makes sense to me, but I still don't understand the - 1. Are you just trying to rind down.

Comment (asottile) [inline scrape.py:368]: -1 is to force some amount of overlap (to counteract the estimation with the other constants such as 111111)

**Pull request merged**